code/index.html

```diff
@@ -16,6 +17,48 @@
       <img src="example-recipe-background.png" alt="Banana Bread">
 
     -->
-    <h1>Hello there! 👋🏼</h1>
+    <img src="./Fotos/Logo_Treuhand.PNG" alt="Logo">
+    
+        
+    <header>
+      <menu>
+        <a href="./index.html">Home</a>
+        <a href="./steuererklärung.html">Steuererklärung</a>
+        <a href="./kmu.html">Services für KMUs</a>
+        <a href="./about.html">Über uns</a>
+        </menu>
+      <h1>Treuhandbüro Tellenbach - Herzlich Willkommen!</h1>
+    </header>
+
+  <div class="Introduction">
```

Comment:
I understand, from the comment on your PR, that this site is still a work in progress. But maybe this divs could be sections to give them a more semantic tag?